weed/storage/volume_tier.go

```diff
@@ -72,6 +73,12 @@ func (v *Volume) LoadRemoteFile() error {
 func (v *Volume) SaveVolumeInfo() error {
 
 	tierFileName := v.FileName(".vif")
+	if v.Ttl != nil {
+		ttlMills := v.Ttl.ToSeconds()
+		if ttlMills > 0 {
+			v.volumeInfo.DestroyTime = uint64(time.Now().Unix()) + v.Ttl.ToSeconds() //生成ec文件开始计算失效时间
```

Comment:
`v.Ttl.ToSeconds()` is repeated again here.

Reply:
You’re right, this place is a bit confusing.